x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/api/ml_models/text_expansion/start_text_expansion_model_api_logic.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ELSER_MODEL_ID } from '../../../../../../common/ml_inference_pipeline';
+import { Actions, createApiLogic } from '../../../../shared/api_logic/create_api_logic';
+import { HttpLogic } from '../../../../shared/http';
+
+export type StartTextExpansionModelArgs = undefined;
```

Comment:
nitpick(non-blocking): Best practice is to specify an empty object rather than undefined.

Reply:
Hm, this is breaking the type checks:
```
/var/lib/buildkite-agent/builds/kb-n2-2-spot-a440796eab9fe2c4/elastic/kibana-pull-request/kibana/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/api/ml_models/text_expansion/create_text_expansion_model_api_logic.ts
--
  | 2023-04-28 11:19:48 EDT | 11:18  error  An empty interface is equivalent to `{}`  @typescript-eslint/no-empty-interface
```

Reply:
@sphilipse I reverted to `undefined` because I couldn't find a way to fix the CI error.

Reply:
Sorry, what you want to do instead is use {} wherever you're using StartTextExpansionModelArgs. But this is a nitpick anyway.